hazelcast/src/test/java/com/hazelcast/util/scheduler/SecondsBasedEntryTaskSchedulerTest.java

```diff
@@ -120,23 +106,25 @@ public void test_rescheduleEntry_postpone() {
     }
 
     @Test
+    // setting system property, not a ParallelTest !
+    @Category(QuickTest.class)
```

Comment:
@mdogan in fact, this test should work even with parallel runner without effecting other tests.
I had added local and thread-local system properties for serial and parallel test runners.
See my previous PR https://github.com/hazelcast/hazelcast/pull/6193